Remove jsafrane from some tests

jsafrane opened the pull request:
I do not know anything about ESIPP nor clouddns and I have never touched these tests. It would be better to assign flakes to someone else.

@bprashanth @quinton-hoole, PTAL. I took your names as authors of these two tests.

k8s-reviewable commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.kubernetes.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/38107)
<!-- Reviewable:end -->

ixdy commented on test/test_owners.csv:
```diff
@@ -520,7 +520,7 @@ k8s.io/kubernetes/federation/apis/federation/install,nikhiljindal,0
 k8s.io/kubernetes/federation/pkg/dnsprovider,sttts,1
 k8s.io/kubernetes/federation/pkg/dnsprovider/providers/aws/route53,cjcullen,1
-k8s.io/kubernetes/federation/pkg/dnsprovider/providers/google/clouddns,jsafrane,1
+k8s.io/kubernetes/federation/pkg/dnsprovider/providers/google/clouddns,quinton,1
```
maybe assign this to @madhusudancs instead.

jsafrane commented:
rebased and used @madhusudancs instead of Quinton. Sorry it took so long.

madhusudancs commented:
Alright, I am signing up for it. LGTM.

Wait for @bprashanth too.

jsafrane committed:
Remove jsafrane from some tests

I do not know anything about ESIPP nor clouddns and I have never touched
these tests. It would be better to assign flakes to someone else.

deads2k commented:
@k8s-bot kops aws e2e test this

deads2k commented:
> Alright, I am signing up for it. LGTM.
> 
> Wait for @bprashanth too.

he gave a conditional lgtm up here https://github.com/kubernetes/kubernetes/pull/38107#discussion_r91187054

deads2k commented:
/lgtm
/approve
